deps/isabelle/Makefile

```diff
@@ -0,0 +1,55 @@
+ISABELLE_VSN:=Isabelle2011-1
+ISABELLE_TGZ:=$(ISABELLE_VSN)_bundle_x86-linux.tar.gz
```

Comment:
This fails in MacOS, and most probably on Windows.

In MacOS, you have to download a `.dmg` file, which is not a `.tar.gz`, and use a different set of commands to extract files from it. IOW, the recipe for `$(ISABELLE_DIR): $(ISABELLE_TGZ)` will have to be completely different. Does `dune` give you access to the OS name, or will you need to make this a `Makefile.in` and tweak `configure`?

Reply:
I have updated the makefile to consider different OS types. Isabelle still doesn't work.

  - On MacOS, I'm getting `poly: Bad CPU type in executable`, I guess the 32 bit support is missing. At least it was needed on Linux for Isabelle to work.
  - On Cygwin, the following DLLs/Versions are missing: `cyggcc_s-1.dll`, `cyggmp-3.dll` and I can't find them for installation in the Cygwin package repository. The old release script was adding `cyggmp-3.dll` to the Isabelle, but the DLL is not committed to the repo; thus, I guess it should fail as well.